src/components/Modal/Modal.tsx

```diff
@@ -119,11 +119,7 @@ export const Modal: React.FunctionComponent<ModalProps> & {
     (evt: React.SyntheticEvent<HTMLIFrameElement>) => {
       const iframe = evt.target as HTMLIFrameElement;
       if (iframe && iframe.contentWindow) {
-        try {
-          setIframeHeight(iframe.contentWindow.document.body.scrollHeight);
-        } catch {
-          setIframeHeight(DEFAULT_IFRAME_CONTENT_HEIGHT);
-        }
+        setIframeHeight(iframe.contentWindow.document.body.scrollHeight || DEFAULT_IFRAME_CONTENT_HEIGHT);
```

Comment:
Is there any reason this needs the fallback `DEFAULT_IFRAME_CONTENT_HEIGHT`. The `if (iframe && iframe.contentWindow)` seems pretty safe and I wondered if it would ever use that value.

Reply:
Oh nice!